File: memory-domain/src/main/java/com/memory/domain/calendar/repository/BaseCalendarEventRepository.java
package com.memory.domain.calendar.repository;

import com.memory.domain.calendar.BaseCalendarEvent;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.repository.NoRepositoryBean;

@NoRepositoryBean
public interface BaseCalendarEventRepository<T extends BaseCalendarEvent> extends JpaRepository<T, Long> {
}

File: memory-domain/src/main/kotlin/com/memory/domain/calendar/repository/BaseCalendarEventRepository.kt
package com.memory.domain.calendar.repository

import com.memory.domain.calendar.BaseCalendarEvent
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.repository.NoRepositoryBean

@NoRepositoryBean
interface BaseCalendarEventRepository<T : BaseCalendarEvent> : JpaRepository<T, Long>
